incubator/command-generate/src/main/java/io/aklivity/zilla/runtime/command/generate/internal/airline/ConfigGenerator.java

```diff
@@ -48,6 +49,7 @@ public abstract class ConfigGenerator
     protected static final String INLINE_CATALOG_TYPE = "inline";
     protected static final String APPLICATION_JSON = "application/json";
     protected static final String VERSION_LATEST = "latest";
+    protected static final Matcher JSON_CONTENT_TYPE = Pattern.compile("^application/(?:.+\\+)?json$").matcher("");
```

Comment:
`Pattern` can be `static` but `Matcher` should be non-static as it is not readonly.

Reply:
OK